tests/phpunit/tests/functions/wpMaybeLoadWidgets.php

```diff
@@ -0,0 +1,31 @@
+<?php
+/**
+ * Tests for wp_maybe_load_widgets function.
+ *
+ * @group functions.php
+ *
+ * @covers ::wp_maybe_load_widgets
+ */#
```

Comment:
```suggestion
 */
```
There was just a stray `#` here